benchmark/CMakeLists.txt

```diff
@@ -11,6 +11,12 @@ if (GINKGO_BUILD_CUDA AND GINKGO_BUILD_HIP AND GINKGO_HIP_PLATFORM MATCHES "hcc"
         "or use `export HIP_PLATFORM=nvcc` in your build environment instead.")
 endif()
 
+function(ginkgo_benchmark_add_tuning_maybe name)
```

Comment:
Why the `_maybe` at the end ?

Reply:
Because it doesn't always add the file, and does it only when needed (thankfully).